Weather App/script.js

```diff
@@ -2,9 +2,23 @@ const apiKey = "da367fc24ba793de8eff595872d99280";
 const apiUrl = "https://api.openweathermap.org/data/2.5/weather?units=metric&q=";
 const searchBox = document.querySelector(".search input");
 const searchBtn = document.querySelector(".search button");
+const searchForm = document.getElementById("searchForm");
 const Icon = document.querySelector(".icon");
 const defaultCity = "Guadalajara";
 
+document.addEventListener("DOMContentLoaded", function () {
+  searchForm.addEventListener("submit", function (event) {
+    event.preventDefault();
+
+    const formData = new FormData(searchForm);
+    const cityName = formData.get("cityName");
+
+    checkWeather(cityName.trim());
+
+    searchForm.reset();
+  });
+});

```

Comment:
@pSapien Add a `submit` listener such that upon the user pressing the enter key, this function is to be called.